### PR TITLE
Add `gr.BrowserState` change event

### PR DESCRIPTION
**Pull request by abidlabs**
Closes: https://github.com/gradio-app/gradio/issues/10243

Run the updated `demo/browserstate/run.py` to verify

### CODE REVIEW

File: js/browserstate/Index.svelte [hunk 1]
```diff
@@ -4,13 +4,19 @@
...
 	export let secret: string;
 	export let default_value: any;
 	export let value = default_value;
 	let initialized = false;
 	let old_value = value;
+	export let gradio: Gradio<{
+		change: never;
+	}>;
+
+	console.log("gradio", gradio);
```
Comment (dawoodkhan82) [inline js/browserstate/Index.svelte:19]: remove?

**Comment by dawoodkhan82** [comment]
Should we add a way to clear the browser state?

**Comment by abidlabs** [comment]
I don't think its necessary @dawoodkhan82, we typically only have a `.clear()` event when there's an actual "X" icon in the UI. So for example, the `gr.Image` component has a `.clear()` event but `gr.Textbox()` does not. In this case, you could listen for the `.change()` event and just check in your function if the value of the BrowserState component is `""`

**Review by dawoodkhan82 — APPROVED** [review]
Tested and lgtm!

**Comment by abidlabs** [comment]
Thanks @dawoodkhan82!

**Pull request merged**